src/coreclr/nativeaot/Bootstrap/main.cpp

```diff
@@ -168,11 +168,12 @@ extern "C" int __managed__Main(int argc, char* argv[]);
 #else
 #define NATIVEAOT_ENTRYPOINT __managed__Startup
 extern "C" void __managed__Startup();
+
+#ifdef TARGET_WASI
```

Comment:
Why is `NATIVEAOT_DLL` dropped here but still included in `InitializeRuntime` ?   If this is intended then I suppose the comment on 175 should be changed?

Reply:
It's not really dropped, it just wasn't needed, since we're in the `#else` part of `#ifndef NATIVEAOT_DLL`:
```cpp
#ifndef NATIVEAOT_DLL
#define NATIVEAOT_ENTRYPOINT __managed__Main
#if defined(_WIN32)
extern "C" int __managed__Main(int argc, wchar_t* argv[]);
#else
extern "C" int __managed__Main(int argc, char* argv[]);
#endif
#else // NATIVEAOT_DLL guaranteed to be defined below
#define NATIVEAOT_ENTRYPOINT __managed__Startup
extern "C" void __managed__Startup();

#ifdef TARGET_WASI
```